src/modules/assertions/require_created_at.ts

```diff
@@ -0,0 +1,21 @@
+// Issue #65, Originally from #55
+// Immediately before merging (IMO, ideally in the merge commit itself) set the created date in the preamble.
+// Creator: SamWilsn
+// Alternative solution: warn author to include created_at date
+
+// to be included when merged in /src/modules/assertions 
+
+import { context, getOctokit } from "@actions/github";
+import {
+  GITHUB_TOKEN,
+  GithubSelf,
+  PR,
+  Review
+} from "src/domain";
+	
+export const getCreatedAt = () => { 
+	const github = getOctokit(GITHUB_TOKEN).rest;
+	 console.warn(`Warning: Please ensure to include in the preamble PR_Created_date: ${context.payload?.pull_request?.created_at}`);
+
+}; 
```

Comment:
This doesn't appear to actually check anything?  Am I misunderstanding something here?

Reply:
This change was reviewed  before (and covers issue 78 (alternative to issue 65)).
just to warn and log created_at. During the issue's discussion it was mentioned that could save time to know created_at date.

Reply:
The logs are *only* looked eat if there is an error, so logging a non-error there won't likely help anything.

Reply:
Ok, so If no created_at found then Log..